### ISSUE TITLE
Which PPO model was trained using allenai/llama-3-tulu-2-8b-uf-mean-rm?

### ISSUE THREAD

**Issue by arunasank** [allenai/open-instruct#380]
It says on https://huggingface.co/allenai/llama-3-tulu-2-8b-uf-mean-rm that he reward model was used during PPO training with the following hyper-parameters, but it's unclear which model is the PPO LLM.Could you please clarify?

> The following hyperparameters were used during PPO training:
>
>    learning_rate: 1e-06
>    total_train_batch_size: 512
>    optimizer: Adam with betas=(0.9,0.999) and epsilon=1e-08
>    lr_scheduler_type: linear cooldown to 1e-05.
>    lr_scheduler_warmup_ratio: 0.03
>    num_epochs: 1.0

**Comment by arunasank** [comment]
cc @vwxyzjn @hamishivi

**Comment by hamishivi** [comment]
Hi @arunasank thanks for the question! Unfortunately we haven't released a model trained with PPO wth that RM, I just trained and released the RM since it was relatively quick and easy (compared to PPO training). I might have a PPO trained model hanging around if that would be useful to release?

**Comment by arunasank** [comment]
Yes, please! :) Thank you for responding.

**Comment by hamishivi** [comment]
Sorry for the wait, but here you go: https://huggingface.co/allenai/tulu-v2.5-llama3-8b-uf-mean-8b-uf-rm

Let me know if there are any issues. This model is the 8b RM llama 3 model described in Table 5 of our [unpacking dpo and ppo paper](https://arxiv.org/abs/2406.09279).
